diff --git a/node_modules/@lit-protocol/auth-browser/src/lib/chains/eth.js b/node_modules/@lit-protocol/auth-browser/src/lib/chains/eth.js
index fb9003c..82a94b9 100644
--- a/node_modules/@lit-protocol/auth-browser/src/lib/chains/eth.js
+++ b/node_modules/@lit-protocol/auth-browser/src/lib/chains/eth.js
@@ -19,8 +19,8 @@ const nacl = require("tweetnacl");
 const buffer_1 = require("buffer");
 const misc_1 = require("@lit-protocol/misc");
 const misc_browser_1 = require("@lit-protocol/misc-browser");
-if (typeof global.Buffer === 'undefined') {
-    global.Buffer = buffer_1.Buffer;
+if (typeof globalThis.Buffer === 'undefined') {
+    globalThis.Buffer = buffer_1.Buffer;
 }
 var WALLET_ERROR;
 (function (WALLET_ERROR) {
